src/components/organisms/FiltersPresetModal/FiltersPresetModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from 'react';
+
+import {Form, Input, Modal, Select} from 'antd';
+
+import {AlertEnum} from '@models/alert';
+
+import {useAppDispatch, useAppSelector} from '@redux/hooks';
+import {setAlert} from '@redux/reducers/alert';
+import {deleteFilterPreset, loadFilterPreset, saveFilterPreset} from '@redux/reducers/main';
+import {closeFiltersPresetModal} from '@redux/reducers/ui';
+
+import * as S from './FiltersPresetModal.styled';
+
+const FiltersPresetModal: React.FC = () => {
```

Comment:
I think I would rather show directly a dropdown with search when the `Load preset` button is clicked.
Alternatively, make the dropdown in this modal to automatically expand and set it to auto focus.